scripts/dockerfiles/Dockerfile.release

```diff
@@ -20,6 +20,9 @@ COPY ["LICENSE", "NOTICE", "/"]
 
 COPY out/amazon-ecs-pause.tar /images/amazon-ecs-pause.tar
 
+# Copy our cni plugins ecs-eni, ecs-ipam and ecs-bridge
+COPY out/cni-plugins /cni-plugins
```

Comment:
If someone is building their own CNI plugin and want to be executed via the ECS Agent, I'd expect it to be mounted at a separate directory so as to not mess up with existing mounts. Having the same path as the default in the Agent makes it less confusing I think.